runtime/context_test.go

```diff
@@ -0,0 +1,28 @@
+package runtime_test
+
+import (
+	"github.com/gengo/grpc-gateway/runtime"
+	"google.golang.org/grpc/metadata"
+
+	"golang.org/x/net/context"
```

Comment:
Standard library packages should be the first group of imports.
https://code.google.com/p/go-wiki/wiki/CodeReviewComments#Imports

Also `"golang.org/x/net/context"` is not a standard package (yet).